site/book/01-getting-started/02-quickstart.md

```diff
@@ -19,7 +19,7 @@ Next, let's quickly view the content of the package:
 
 ```shell
 $ kpt pkg tree
-Package nginx:
+Package "nginx":
```

Comment:
There are more references to tree in the docs. You can do a grep in the repo, or search through the UI: https://kpt.dev

Reply:
The best way to fix this is to add verify guides mark up for the guides in book similar to reference docs. https://github.com/GoogleContainerTools/kpt/blob/next/site/reference/pkg/get/README.md#examples

You can do it as your next task.

Reply:
I found 4 references of tree command in the book apart from command reference using the grep command. I manually checked the output and updated the docs accordingly.